torch/distributions/__init__.py

```diff
@@ -28,6 +28,21 @@
     next_state, reward = env.step(action)
     loss = -m.log_prob(action) * reward
     loss.backward()
+
+Another way to implement these stochastic/policy gradients would be to use the
+reparameterization trick from :meth:`~torch.distributions.Distribution.rsample`
+method, where the parameterized random variable can be defined as a parameterized
+deterministic function of a parameter-free random variable. The reparameterized sample
+is required to be differentiable. The code for implementing the pathwise estimation would
+be as follows::
+
+    probs = policy_network(state)
+    # NOTE: this is equivalent to what used to be called multinomial
+    m = Categorical(probs)
```

Comment:
`Categorical` is not reparameterized; we should use an example with `.has_rsample` here, e.g.

```py
params = policy_network(state)
action = Normal(**params).rsample()
next_state, reward = env.step(action)  # assume enf.step is differentiable
loss = -reward
loss.backward()
```